### PR TITLE
initial com

### PR DESCRIPTION
**Pull request by MaksymKos**
Знімок екрана 2023-07-27 о 14 43 57

need some help

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,107 +1,33 @@
...
+}
 
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
+function getCommentsById(id) {
+  return commentsFromServer.filter(comment => comment.postId === id)
+      || null;
```
Comment (IvanFesenko) [inline src/App.jsx:15]: null is redudant - filter always return an array
```suggestion
  return commentsFromServer.filter(comment => comment.postId === id);
```

File: src/App.jsx [hunk 2]
```diff
@@ -1,107 +1,33 @@
...
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
+export const App = () => (
+  <section className="App">
+    <h1 className="App__title">Static list of posts</h1>
+    <PostList
+      posts={postsFromServer}
```
Comment (IvanFesenko) [inline src/App.jsx:28]: That's might help with your error.
```suggestion
      posts={posts}
```

File: src/components/PostInfo/PostInfo.jsx [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```
Comment (IvanFesenko) [inline src/components/PostInfo/PostInfo.jsx:13]: User might be null - add check if post.user exists

File: src/App.jsx [hunk 4]
```diff
@@ -1,107 +1,33 @@
...
+      comments={commentsFromServer}
+      users={usersFromServer}
```
Comment (IvanFesenko) [inline src/App.jsx:30]: Redudant props - not used
```suggestion

```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Check comments. Next time if you have have some questions - ask them in fe_chat

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**